tests/aead_tests.rs

```diff
@@ -509,6 +509,38 @@ fn test_aead_key_debug() {
     );
 }
 
+#[test]
+#[cfg_attr(target_arch = "wasm32", wasm_bindgen_test)]
```

Comment:
The wasm_bindgen_test thing here shouldn't be necessary anymore because of what was done in PR #1252.